x/crosschain/module.go

```diff
@@ -176,7 +175,14 @@ func (am AppModule) ExportGenesis(ctx sdk.Context, cdc codec.JSONCodec) json.Raw
 func (AppModule) ConsensusVersion() uint64 { return 2 }
 
 // BeginBlock executes all ABCI BeginBlock logic respective to the crosschain module.
-func (am AppModule) BeginBlock(_ sdk.Context, _ abci.RequestBeginBlock) {}
+func (am AppModule) BeginBlock(ctx sdk.Context, _ abci.RequestBeginBlock) {
+	if ctx.BlockHeight() == 200 {
```

Comment:
Why using `200`? I think we could define a constant variable for this value `HeightTestWhitelistERC20`

Reply:
just for smoketest. We have quite a few magic numbers for smoketest that is probably ok.